SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/BaseData.h

```diff
@@ -350,7 +319,7 @@ class SOFA_CORE_API BaseData : public DDGNode
     std::string m_name;
 
     /// Parent Data
-    SingleLink<BaseData,BaseData,BaseLink::FLAG_STOREPATH|BaseLink::FLAG_DATALINK|BaseLink::FLAG_DUPLICATE> parentData;
+    DataLink<BaseData> parentData;
```

Comment:
so If I got it... all this work for this small line?

Reply:
Yes :)